### PR TITLE
[WFCORE-4748] Allow the MODULE_OPTS to be set

### PR DESCRIPTION
**Pull request by jamezp**
**NOTE:** This is not yet complete, but I wanted to get a CI run to test the different environments.

The first commit just upgrades the wildfly-maven-plugin and changes the property name is it was rather confusing with the `version.org.wildfly.plugins` property.

The second commit exposes the `MODULE_OPTS` environment variable and adds `jboss-modules.jar` as an agent if there is an agent defined in the options. A documentation PR will be required upstream before this can be merged.

https://issues.redhat.com/browse/WFCORE-4748
https://issues.redhat.com/browse/WFCORE-4897

### CODE REVIEW

File: core-feature-pack/src/main/resources/content/bin/standalone.conf.bat [hunk 1]
```diff
@@ -82,3 +82,6 @@ rem # Uncomment and edit to use a custom java.security file to override all the
 rem set "JAVA_OPTS=%JAVA_OPTS% -Djava.security.properties==C:\path\to\custom\java.security"
 
 :JAVA_OPTS_SET
+
+rem # Uncomment to add a Java agent
+rem set "MODULE_OPTS=-javaagent:agent.jar"
```
Comment (yersan) [inline core-feature-pack/src/main/resources/content/bin/standalone.conf.bat:87]: Just for the record regarding these script changes; we are going to need to adapt the specific scripts used for the product conversion as well

Comment (jamezp) [reply core-feature-pack/src/main/resources/content/bin/standalone.conf.bat:87]: I guess I didn't realize we had custom scripts to those. Can you point me to them so I can have a look?

Comment (yersan) [reply core-feature-pack/src/main/resources/content/bin/standalone.conf.bat:87]: Hello James, sure: 
https://github.com/jbossas/wildfly-to-product/commits/wf-to-cd

Take a look at "Community to EAP conversion -- Bring in EAP-specific scripts "

Comment (jamezp) [reply core-feature-pack/src/main/resources/content/bin/standalone.conf.bat:88]: Done https://github.com/jbossas/wildfly-to-product/pull/46

**Comment by jamezp** [comment]
@wildfly/mergers-full Just an FYI the last change which triggers the rebuild will have no affect on testing and this was previously green. The only two changes were a change to the attribute description and a change to the `DomainCommandBuilder`.

**Comment by jamezp** [comment]
This has been rebased and updated for WildFly Core 13.

**Comment by jmesnil** [comment]
@jamezp sorry (again!), this PR also needs a rebase after #4234. The impacted files are now in https://github.com/wildfly/wildfly-core/tree/master/core-feature-pack/common/src/main/resources/content/bin

**Comment by jamezp** [comment]
Not a problem at all @jmesnil. I've rebased it and removed the label.

**Comment by jmesnil** [comment]
@jamezp thanks

**Pull request merged**